### ISSUE TITLE
Plugin removed from Chrome Web Store

### ISSUE THREAD

**Issue by Wiethoofd** [Simego/TwitchEmotes4WhatsApp#2]
The [plugin page](https://chrome.google.com/webstore/detail/nokdilphncflibihloahbppnfmefmpfc) in the Chrome Web Store is returning a 404.

There is an [archived](https://crx.dam.io/ext/nokdilphncflibihloahbppnfmefmpfc.html) version available. But it seems that the plugin has been taken down and also does not work anymore.

---

Note: If you still have a craving for Twitch emotes in WhatsApp and don't mind them everywhere, you could try [this Global Twitch Emotes plugin](https://chrome.google.com/webstore/detail/global-twitch-emotes/pgniedifoejifjkndekolimjeclnokkb) instead.

**Comment by Simego** [comment]
Yeah it was taken down unfortunately by Whatsapp but like you said there is the global emotes plugin available :)
